DllImportGenerator/DllImportGenerator/MarshallingAttributeInfo.cs

```diff
@@ -632,21 +636,26 @@ CountInfo CreateCountInfo(AttributeData marshalUsingData, ImmutableHashSet<strin
 
             if (methods == SupportedMarshallingMethods.None)
             {
-                _diagnostics.ReportConfigurationNotSupported(attrData, "Native Type", nativeType.ToDisplayString());
+                _diagnostics.ReportInvalidMarshallingAttributeInfo(
+                    attrData,
+                    isContiguousCollectionMarshaller ?
```

Comment:
nit: I prefer the `?` on the next line (so things following `?`/`:` line up nicely)

Reply:
I had it like that, but I wasn't sure how we preferred it. I'll change it back.